test/Microsoft.DotNet.CommandFactory.Tests/GivenAProjectDependencyCommandResolver.cs

```diff
@@ -30,8 +30,7 @@ public void ItReturnsACommandSpecWhenToolIsInAProjectRef()
             MSBuildTestProjectInstance =
                 TestAssets.Get("TestAppWithProjDepTool")
                     .CreateInstance()
-                    .WithSourceFiles()
-                    .WithRestoreFiles();
+                    .WithSourceFiles();
```

Comment:
Are there breaking changes in the new stage 0?

Reply:
I don't think so, I'm still trying to figure out why some of the tests are failing with a missing packages directory.